docs/Configuration.md

```diff
@@ -31,32 +30,118 @@ SQLite            | `SQLite`
 
 ### user
 
+**Deprecated.** Use MYSQL.user or POSTGRESQL.user instead.
+
 The name of the user with sufficient permission to access the database.
 
 Ignored by the SQLite database engine.
 
```

Comment:
I think the four lines above can be removed to keep it shorter. The references to the new configuration makes it very clear.

Reply:
I like to keep the documentation of deprecated APIs even if we provide a migration path away from them. But sure, I'll remove it this time. I think it's a good thing that we stop and think about how we deprecate things.